package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "xhtml-purifier",
-	"version": "0.3.1",
+	"version": "0.3.2",
```

Comment:
so in that case, we should be bumping the major version number to ```1.0.0```. or, we could make the changes backwards-compatible, and bump to ```0.4.0```. either way works